Quadriga/src/main/java/edu/asu/spring/quadriga/rest/open/SearchConceptController.java

```diff
@@ -47,8 +49,13 @@
 
         List<JSONObject> jsonResults = new ArrayList<JSONObject>();
 
+        CopyOnWriteArrayList<ConceptEntry> cowConceptList = new CopyOnWriteArrayList<>(conceptList);
+        Iterator<ConceptEntry> conceptIterator = cowConceptList.iterator();
```

Comment:
why do you use a CopyOnWriteArrayList here? conceptList is doesn't get modified or used by several threads, does it?

Reply:
Am not sure about this. I continued working on an existing branch...This piece of code was already there....Will check.